Update keyboard.json

iliorik opened the pull request:
change button coordinates on right half

<!--- Provide a general summary of your changes in the title above. -->

<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

## Description

<!--- Describe your changes in detail here. -->

## Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

zvecr reviewed (changes requested):
While this may fix the display issues, the order still does not match that expected by the keymap. The whole right hand is mostly reversed, with the thumb clusters being quite out of order.

The fact its so wrong suggests this and the previous submittion has not actually been tested on hardware.

iliorik commented:
I'm already typing on this keyboard. How could I not test it?

zvecr commented:
![image](https://github.com/user-attachments/assets/937cb16b-a515-4896-a4fc-03944da0169e)
Still incorrect. You can validate this by previewing on <https://config.qmk.fm>, by using `ctrl+shift+i` to load the keyboard.json and enabling "fast input" in settings menu.

iliorik commented:
![shot-2025-01-03_23-58-41](https://github.com/user-attachments/assets/c238eb89-ab67-47f5-8f81-43bfe4c9f97f)
if I usе `ctrl+shift+i` to load the keyboard.json, I get this message. What am I doing wrong?

sigprof commented:
You have a trailing comma: `"mousekey": true,` — this is not allowed in JSON.  QMK build scripts read the `keyboard.json` files as Hjson, which allows trailing commas and some other extensions, but config.qmk.fm needs pure JSON.

iliorik commented:
I've fixed the layout. i think all ok. (my keyboard config: https://github.com/iliorik/qmk_configurator/blob/master/public/keymaps/h/handwired_dactyl_manuform_5x8_default.json)
![shot-2025-01-04_14-31-07](https://github.com/user-attachments/assets/1da4db2d-e323-43ff-bfe2-355b1a46d383)
![shot-2025-01-04_14-31-37](https://github.com/user-attachments/assets/538c1f20-925e-414c-ab34-38b4cee0f854)

iliorik commented:
I analyzed and adjusted the order of the keys to make sure that it matches the expected physical location. In particular, I have adjusted the order of the right keys and thumb clusters to align it with the expectations of the keyboard layout.
 I am currently typing on this keyboard and have made sure that the changes are working properly. However, I would appreciate any additional testing or validation that you, @zvecr, can provide to make sure that everything is done correctly.

iliorik commented:
Thank you for pointing it out!
I have removed the trailing comma from the mousekeys entry.
Please let me know if there are any other issues.

zvecr commented:
> Is that correct?

Nope, as you have now reversed the left hand portion of the layout, effectively breaking it. This can be easily noted if it were tested on hardware.

You would need to keep the `matrix_pins.cols` as it previous was, and merge in the following
```json
    "split": {
        "matrix_pins": {
            "right": {
                "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2", "B6"]
            }
        }
    }
 ```

iliorik commented:
Hello @zvecr,
I have corrected the code and tested it on the keyboard. This code works. Can you approve the changes for the merge?

iliorik commented:
Hello @daskygit ,
Can you review my code?
